homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -48,3 +48,14 @@ class AutomowerControlEntity(AutomowerBaseEntity):
     def available(self) -> bool:
         """Return True if the device is available."""
         return super().available and self.mower_attributes.metadata.connected
+
+
+class AutomowerErrorConfirmEntity(AutomowerControlEntity):
```

Comment:
This class seems quite specific to the button entity so why not include it there instead of creating it separately here to inherit from?